viadot/sources/cloud_for_customers.py

```diff
@@ -0,0 +1,257 @@
+import re
+import requests
+import pandas as pd
+
+from copy import deepcopy
+from typing import Any, Dict, List, Optional
+from pydantic import BaseModel
+from urllib.parse import urljoin
+from viadot.exceptions import CredentialError
+
+from ..utils import handle_api_response
+from ..config import get_source_credentials
+from .base import Source
+
+
+class CloudForCustomersCredentials(BaseModel):
+    username: str  # eg. username@{tenant_name}.com
+    password: str
```

Comment:
maybe we should use SecretStr here? can you give me your opinion

Reply:
Great! will prevent some security issues like logging sensitive data.